SmartEVSE2.X/EVSE.c

```diff
@@ -135,9 +135,9 @@ unsigned int CalcCurrent();
 #pragma	config BORV = 285, BOREN = ON, PWRTEN = ON
 #pragma	config WDTPS = 2048, WDTEN = OFF                                        // WDT timeout
 #pragma config CCP2MX = PORTB3, PBADEN = OFF, CCP3MX = PORTC6                   // PortB digital IO
-#pragma config HFOFST = OFF, T3CMX = PORTB5, P2BMX = PORTC0, MCLRE = INTMCLR
+#pragma config HFOFST = OFF, T3CMX = PORTB5, P2BMX = PORTC0, MCLRE = EXTMCLR    // MCLR Pin Enable bit (MCLR pin enabled, RE3 input pin disabled)
```

Comment:
There is no Pull-up on the reset pin, so enabling this pin might trigger a sudden reset in noisy environments.

Reply:
The fail safe clock monitor can be enabled by setting the FCMEN bit in the
CONFIG1H Configuration register. But the config registers are not writable in a firmware update. While it might be a good idea to have a fail safe, i have never seen a xtal oscillator fail on a module.

Reply:
Ok, agree about the pull-up.
The fail-safe: no, have never seen it fail either, but using this setting allows the code to work both on PCB with and without Xtal. (It seems to me SmartEVSE does not strictly need the precision of an Xtal vs. the INTOSC, does it?)

Reply:
I'll test it on a module soon. The 1kHz CP signal may vary +-2% according to the J1772 spec.
The HFINTOSC is factory calibrated to +-2% between 0°C and +60°C, so it should at least work most of the time.
![HFINTOSC](https://user-images.githubusercontent.com/1167021/114239307-32177e80-9986-11eb-9a6b-285b4a79cad1.png)

Reply:
Tested it on a module, switchover seems to be working fine.
Measured frequency is 1002 Hz (at 20C), so pretty close.
at -30C, the frequency changed to 998 Hz, still OK.
at +55C the frequency changed to 1001 Hz